components/brave_shields/content/browser/BUILD.gn

```diff
@@ -28,8 +28,6 @@ static_library("browser") {
     "ad_block_subscription_service_manager_observer.h",
     "adblock_stub_response.cc",
     "adblock_stub_response.h",
-    "base_brave_shields_service.cc",
-    "base_brave_shields_service.h",
```

Comment:
this removal is not related to strip_rust_symbols, right? Or I am missing something

Reply:
oops, I think I accidentally had some overlap between two PRs I have open

Reply:
yep, merged here https://github.com/brave/brave-core/pull/30503/files, I'll rebase